src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="width100 fl marginT20">
-  <div id="workItemQuickAdd_container" class="width100 fl">
+<div class="width100 fl quickAdd_container" id="workItemQuickAdd_container" >
+  <!--<div id="workItemQuickAdd_container" class="width100 fl">
```

Comment:
There is no need to keep a commented version if you don't plan on going back to it. Git history is usually enough to find the way it worked in the past.